nixos/doc/manual/from_md/development/writing-documentation.chapter.xml

```diff
@@ -1,4 +1,4 @@
-<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="sec-writing-documentation">
+<chapter xmlns="http://docbook.org/ns/docbook"  xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude" xml:id="sec-writing-documentation">
```

Comment:
Not sure how this change relates script changes.

Reply:
The from_md xml files need to be updated whenever the script changes, and in this case xi:include is mentioned in the Markdown file, so this changed.